docs/next.config.js

```diff
@@ -118,7 +118,7 @@ module.exports = withNextPluginPreval({
     ];
     const defaultRemarkPlugins = [
       require('remark-code-import'),
-      require('remark-gfm'),
+      import('remark-gfm'),
```

Comment:
Why did this change? Seems odd to have both `require` and `import` statements. But if it works 🤷

Reply:
remark-gfm v3 complains about `require`, so had to change to `import`

Reply:
can we change them both to import?

Reply:
I'm not sure if "remark-code-import" can be changed since its version is older